languages/python/app/Main.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Main where
+
+import qualified Data.ByteString as B
+import qualified TreeSitter.Importing as TS
+import TreeSitter.Python
+
+parseByteString :: B.ByteString -> IO (Maybe Module)
+parseByteString = TS.parseByteString tree_sitter_python
+
+main :: IO ()
+main = do
+  parseByteString "from __future__ import absolute_import" >>= print
```

Comment:
We can 🔥 this file along with the `python` executable in the tree-sitter-python.cabal file.